CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## 1.0.0b6 / 2021-02-16
+
+* [Added] Add profile_metrics_query properties to dashboard widget requests. See [#283](https://github.com/DataDog/datadog-api-client-python/pull/283).
+* [Added] Add profile_metrics_query properties to dashboard widget requests. See [#282](https://github.com/DataDog/datadog-api-client-python/pull/282).
```

Comment:
Whats the difference on these two?

Reply:
We can remove 283, looks like its client regeneration. Only thing updated was timestamp.